src/gamebryo/gamebryomoddatacontent.cpp

```diff
@@ -0,0 +1,91 @@
+#include "gamebryomoddatacontent.h"
+
+#include <scriptextender.h>
+
+#include "gamegamebryo.h"
+
+GamebryoModDataContent::GamebryoModDataContent(GameGamebryo const* gamePlugin) : 
+  m_GamePlugin(gamePlugin), m_Enabled(CONTENT_MODGROUP + 1, true) { }
+
+std::vector<GamebryoModDataContent::Content> GamebryoModDataContent::getAllContents() const {
+  static std::vector<Content> GAMEBRYO_CONTENTS{
+    {CONTENT_PLUGIN,    QT_TR_NOOP("Game Plugins (ESP/ESM/ESL)"), ":/MO/gui/content/plugin"},
```

Comment:
remove the ESP/ESM/ESL as it's too long for filters view.

Reply:
Maybe remove the "Game" instead? `Plugins (ESP/ESM/ESL)`?